src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringInterpolation.cs

```diff
@@ -23,7 +20,7 @@ private BoundExpression RewriteInterpolatedStringConversion(BoundConversion conv
             // the arguments first in this situation because we do not know what conversions will be
             // produced for the arguments until after we've done overload resolution. So we produce the invocation
             // and then lower it along with its arguments.
-            var result = _factory.StaticCall(stringFactory, "Create", expressions.ToImmutableAndFree(),
+            var result = _factory.StaticCall(BinderFlags.None, stringFactory, "Create", expressions.ToImmutableAndFree(),
```

Comment:
Can you please make an overload of `StaticCall` that omits the `BinderFlags` argument for clarity of these call sites?